api/src/main/java/org/pmiops/workbench/billing/BillingGarbageCollectionService.java

```diff
@@ -114,10 +112,13 @@ private GarbageCollectedProject recordGarbageCollection(
         () -> new Timestamp(clock.instant().toEpochMilli()));
     billingProjectBufferEntryDao.save(entry);
 
-    return gc.toGarbageCollectedProject();
+    log.info(
```

Comment:
Is there any thought to a queryable table for this information, like `project_gcp_id, owner, lifecycle_stage?

Also, if you want these values to be a separate (searchable) field, you can use the Logging client with a jsonPayload.